src/pgcopy/src/copy.proto

```diff
@@ -0,0 +1,32 @@
+// Copyright Materialize, Inc. and contributors. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+syntax = "proto3";
+
+package mz_pgcopy.copy;
+
+message ProtoCopyFormatParams {
+    oneof kind {
+        ProtoCopyTextFormatParams text = 1;
+        ProtoCopyCsvFormatParams csv = 2;
+    }
+}
+
+message ProtoCopyTextFormatParams {
+    string null = 1;
+    string delimiter = 2;
```

Comment:
> This must be a single one-byte character.

According to the copy pg docs.

Reply:
Yeah I was planning to refactor it https://github.com/MaterializeInc/materialize/pull/25521/files#diff-0b9d991a180a6d2826919d99dedc6547e51f18246dc8c1534b80a766fd687ad4R464. 

Let me do the proto here correctly. The other refactor turned out to be not trivial because I was running into borrow-checker issues

Reply:
Protobuf does not have anything smaller than a uint32 to store a byte/u8, so changing this to uint32 here.

Reply:
Updated (and changed the delimiter to u8  as well)